tutor/src/components/reference-book/reference-book.cjsx

```diff
@@ -18,7 +18,7 @@ MENU_VISIBLE_BREAKPOINT = 1350
 module.exports = React.createClass
   displayName: 'ReferenceBook'
 
-  mixins: [ResizeListenerMixin, ChapterSectionMixin]
+  mixins: [ResizeListenerMixin]
```

Comment:
`sectionFormat` is being used from `ChapterSectionMixin`, ref book errors without it